sample_reefperf_config/sample_app_deployment.json

```diff
@@ -2,19 +2,25 @@
     "app-deployment": {
         "nodes": [
             {
-                "name": "cache",
```

Comment:
it was almost good, but it is really about node types and should be indexed by name. Otherwise sooner or later someone will put two entries with the same in the list...